dynomitemanager-core/src/main/java/com/netflix/dynomitemanager/storage/RedisStorageProxy.java

```diff
@@ -606,17 +631,18 @@ public void updateConfiguration() throws IOException {
                 if (line.startsWith("#")) {
                     isComment = true;
                     String withoutHash = line.substring(1);
-                    if (!withoutHash.matches(REDIS_CONF_SAVE_SCHEDULE) &&
-                            !withoutHash.matches(REDIS_CONF_UNIXSOCKET) &&
-                            !withoutHash.matches(REDIS_CONF_UNIXSOCKETPERM)) {
+                    if (!withoutHash.matches(REDIS_CONF_SAVE_SCHEDULE) && !withoutHash.matches(REDIS_CONF_UNIXSOCKET)
```

Comment:
why did you make this change, it is a common practice to have each condition on a separate line

Reply:
This was automatically done by Java convention that Eclipse provides. I tent to do that when I run code because it structures the new code and therefore affects every other part of the same class.